apps/nextjs/src/app/(main)/page.tsx

```
@@ -10,16 +12,29 @@
 
 export default function Home() {
   return (
-    <div className="min--h-screen flex flex-col items-center justify-between">
-      <PartneredOrganizations />
-      <FrontPageText />
-    </div>
+    <>
+      <section
+        role="region"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<section>


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)